stable/drupal/templates/deployment.yaml

```diff
@@ -19,15 +19,28 @@ spec:
         app: {{ template "drupal.fullname" . }}
         chart: {{ template "drupal.chart" . }}
         release: "{{ .Release.Name }}"
+{{- if or .Values.podAnnotations .Values.metrics.enabled }}
```

Comment:
`podAnnotations` is not defined in `values.yaml` (and `values-production.yaml` if it exists).

Can you add it and also include the new variable in the README?